src/common/config/decrypt.tsx

```diff
@@ -70,22 +70,31 @@ export const decryptWalletOrSigner = async (
     const connectedWallet = await decryptedWallet.connect(provider);
     return connectedWallet;
   } else {
-    const kmsCredentials = {
-      accessKeyId: config.wallet.accessKeyId, // credentials for your IAM user with KMS access
-      secretAccessKey: password, // credentials for your IAM user with KMS access
-      region: config.wallet.region,
-      keyId: config.wallet.kmsKeyId,
-    };
+    switch (config.wallet.type) {
+      case "ECC_SECG_P256K1":
```

Comment:
just AWS KMS, we’re not trying to distinguish between different kinds of aws kms (the other algos cannot be used for eth wallets anyway)
we’re trying to distinguish between different types of wallet configuration like say aws kms, private key, or mnemonic or ledger nano etc

Reply:
ahhhhhhhhh i think i misunderstood previously, okay will make the changes.